src/Meilisearch/TaskInfo.cs

```diff
@@ -28,21 +28,21 @@ public class TaskInfo
         /// <summary>
         /// The unique sequential identifier of the task.
         /// </summary>
-        public int Uid { get; }
+        public int TaskUid { get; }
 
         /// <summary>
         /// The unique index identifier.
         /// </summary>
-        public string IndexUid { get; }
+        public string? IndexUid { get; }
 
         /// <summary>
         /// The status of the task. Possible values are enqueued, processing, succeeded, failed.
         /// </summary>
         public TaskInfoStatus Status { get; }
 
         /// <summary>
-        /// The type of task. Possible values are indexCreation, indexUpdate, indexDeletion, documentAddition,
-        /// documentPartial, documentDeletion, settingsUpdate, clearAll.
+        /// The type of task. Possible values are indexCreation, indexUpdate, indexDeletion,
```

Comment:
I strongly believe you can remove this comment because all the possible types are already described in line 90, and that line is the only that matters :)